### DISCUSSION TITLE
Recosanté unavailable ?

### DISCUSSION THREAD

**Discussion by CelianGdfrd** [breezy-weather/breezy-weather#1010] [Q&A] [answered]
Glad to see Recosanté is enabled, it's a great service.

But I'm not sure why there is no way for me to select it in sources. I'm using Standard flavour. In pollens, air quality, it doesn't show in the list of available sources, even though APIs are entered in settings.

Thanks

**Comment by papjul** [comment, marked as answer]
Salut, you can only use it for locations in France. We only support Recosanté as Pollen source, not air quality which lacks concentration information.

**Comment by CelianGdfrd** [comment]
Thank you very much! Once "Current location" saw that I'm in France, Recosanté became available for pollens. Hopefully they will be soon for pollution.

**Reply by papjul** [reply]
No, Recosanté only has indices, it doesn't (and will probably never) have pollutant concentrations. It is mandatory so that we can calculate our universal air quality index and give you accurate recommandations, rather than using a France-specific air quality index that won't match other locations air quality indexes.

Learn more: https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md#current-air-quality

FYI, Open-Meteo has support for air quality in Europe based on Copernicus using data from different partners in each country.

**Comment by CelianGdfrd** [comment]
Wonderful! Merci for all those explanations :)
